environment_setup/install_clion.sh

```diff
@@ -4,12 +4,56 @@ echo "================================================================"
 echo "Installing CLion"
 echo "================================================================"
 
-# Clion can be installed through snapcraft
-# https://snapcraft.io/clion
-sudo apt-get install snapcraft
-sudo snap install clion --classic
+clion_version="2019.2.2"
+usr_location="/usr/local/bin/clion"
+
+if [ -e ${usr_location} ]
+then
+        echo "================================================================"
+        echo "CLion is already installed"
+        echo "================================================================"
+else
+	# Download clion
+        wget -O ~/Downloads/CLion-${clion_version}.tar.gz "https://download-cf.jetbrains.com/cpp/CLion-${clion_version}.tar.gz"
```

Comment:
`/tmp` is probably the "more correct" folder for this, generally that's where you stick anything you don't want

Reply:
done